evm/gateway.js

```diff
@@ -28,7 +28,8 @@ const {
 const { addBaseOptions } = require('./cli-utils');
 const { getWallet, signTransaction } = require('./sign-utils');
 
-const IGateway = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IAxelarGateway.json');
+const AmplifierGateway = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IAxelarGateway.json');
```

Comment:
this interface only supports messaging. you have to use the interface corresponding to the actual amplifier gateway

Reply:
aha, thanks for pointing it out. updated.